Source/TimeIntegration/ERF_slow_rhs_pre.cpp

```diff
@@ -298,8 +298,8 @@ void erf_slow_rhs_pre (int level, int finest_level,
             const Array4<Real>& pptemp_arr = pprime.array();
             ParallelFor(gbx, [=] AMREX_GPU_DEVICE (int i, int j, int k) noexcept
             {
-                //if (cell_data(i,j,k,RhoTheta_comp) < 0.) printf("BAD THETA AT %d %d %d %e %e \n",
-                //    i,j,k,cell_data(i,j,k,RhoTheta_comp),cell_data(i,j,k+1,RhoTheta_comp));
+                if (cell_data(i,j,k,RhoTheta_comp) < 0.) printf("BAD THETA AT %d %d %d %e %e \n",
+                    i,j,k,cell_data(i,j,k,RhoTheta_comp),cell_data(i,j,k+1,RhoTheta_comp));
```

Comment:
Sycl CI may have issues with `printf` on the gpu
```suggestion
                if (cell_data(i,j,k,RhoTheta_comp) < 0.) AMREX_DEVICE_PRINTF("BAD THETA AT %d %d %d %e %e \n",
                    i,j,k,cell_data(i,j,k,RhoTheta_comp),cell_data(i,j,k+1,RhoTheta_comp));
```

Reply:
Got it, I'll make that change now